modules/resource-logs-viewing-cli-console.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * logging/resource-logs-viewing.adoc
+
+[id="resource-logs-viewing-cli-console_{context}"]
+= Viewing resource logs
+
+You can view the log for various resources in the {product-title} CLI and web console. Logs read from the tail, or end, of the log.
+
+.Prerequisites
+
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+.Procedure 
+
+To view resource logs:
+
+* Using the web console
+
+. In the {product-title} console, navigate to *Workloads* -> *Pods* or navigate to the pod through the resource you want to investigate. 
++
+[NOTE]
+====
+Some resources, such as builds, do not have pods to query directly. Locate the *Logs* link on the *Details* page.
+====
+
+. Select a project from the drop-down menu.
+
+. Click the pod you want to investigate.
```

Comment:
Would "select" make more sense here rather than "click"?

Reply:
"Select", to me suggests picking a checkbox or the like. Here. you click the name of the pod to access the details page. Maybe I should say just that! WDYT?